Model/Quote/GetCartLineItems.php

```diff
@@ -107,6 +108,7 @@ private function getLineItem(CartItemInterface $item): array
             'requires_shipping' => !$item->getProduct()->getIsVirtual(),
             'line_item_key' => (string)$item->getId(),
             'price' => $this->getLineItemPrice($item),
+            'vendor' => '',
```

Comment:
Note: Out of the box Magento doesn't have a vendor field for products so setting this to an empty string.